catalog/core/catalog-core-standardframework/src/main/java/org/codice/ddf/catalog/resource/cache/ResourceCache.java

```diff
@@ -45,6 +46,14 @@
    */
   Optional<Resource> get(Metacard metacard, ResourceRequest resourceRequest);
 
+  /**
+   * Put a resource into the cache based on the given {@link Metacard} and {@link ResourceResponse}.
+   *
+   * @param metacard metacard that corresponds to the resource that was requested
+   * @param resourceResponse response object associated with the {@link Metacard} provided
+   */
+  void put(Metacard metacard, ResourceResponse resourceResponse);
+
```

Comment:
:exclamation: a cache should follow a key/value pattern, so I'd expect the `put()` method to accept the same type that `get()` returns. In this case, I think implementers of this API will expect `put()` to accept a `Resource` since that's what `get()` returns.

Reply:
Fixed